src/rachanaGJun20/PlindromeString.java

```diff
@@ -0,0 +1,41 @@
+package rachanaGJun20;
+
+import java.util.Scanner;
+
+public class PlindromeString {
+	//method will check whether given string is Palindrome
+	boolean isStringPlindrome(String str) {
+		String revstr = "";
+		String givenString = str;
+
+		for (int i = str.length() - 1; i >= 0; i--) {
+			revstr = revstr + str.charAt(i);
+		}
+		if (givenString.equals(revstr))
```

Comment:
Suggestion 2: use equalsIgnoreCase

Reply:
I think , In reversing a string and checking for palindrome we should have to use string as it is in the given format.
we can not use equalsIgnoreCase here I think.